browser_test/svg/svg_clock.js

```diff
@@ -0,0 +1,80 @@
+import { createRenderers } from '../../benchmark/renderers';
+
+const $container = document.querySelector('#container');
+
+const rowsCount = 30;
+const colsCount = 30;
+const tpl = (date, html) => html`
+  <div id="clock-container">
+    <svg id="clock" viewBox="0 0 100 100">
+      <circle id="face" cx="50" cy="50" r="45"/>
+      <g id="hands">
+        <rect id="hour" x="47.5" y="12.5" width="5" height="40" rx="2.5" ry="2.55" transform="rotate(${30 * (date.getHours() % 12) + date.getMinutes() / 2} 50 50)" />
+        <rect id="min" x="48.5" y="12.5" width="3" height="40" rx="2" ry="2" transform="rotate(${date.getMinutes() * 6} 50 50)" />
+        <line id="sec" x1="50" y1="50" x2="50" y2="16" transform="rotate(${date.getSeconds() * 6} 50 50)" />
+      </g>
+    </svg>
+  </div>
+`;
+
```

Comment:
Will it be supporting CSS in the SVG? like as follows:

```svg
<svg width="1800px" height="826px" viewBox="0 0 1800 826" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <style>
    /* <![CDATA[ */
    #logo {
        animation: fade 2s ease 0s infinite alternate;
    }
    #background {
        animation: fade-reverse 1s ease 0s infinite alternate;
    }
    @keyframes fade {
        0% { opacity: 0.3; }
        100% { opacity: 0.8; }
    }
    @keyframes fade-reverse {
        0% { opacity: 0.8; }
        100% { opacity: 0.3; }
    }
    /* ]]> */
    </style>
    <defs>
        <rect id="path-1" x="0" y="0" width="410" height="410" rx="15"></rect>
    </defs>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <rect id="background" fill-opacity="0.572209013" fill="#D8D8D8" x="0" y="0" width="1800" height="826"></rect>
        <g id="favicon" transform="translate(695.000000, 208.000000)">
            <mask id="mask-2" fill="white">
                <use xlink:href="#path-1"></use>
            </mask>
            <use id="Mask" fill-opacity="0.0482620018" fill="#000000" fill-rule="nonzero" xlink:href="#path-1"></use>
            <g mask="url(#mask-2)" id="logo" opacity="0.586503623">
                <g>
                    <rect id="bg" fill="#55ACAA" fill-rule="nonzero" x="0" y="0" width="410" height="410"></rect>
                    <polygon id="T" fill="#414142" fill-rule="nonzero" points="129.941154 0 280.955494 0 410 0 410 111.291407 274.056023 111.291407 274.056023 195.401574 274.056023 307.564352 274.056023 410 136.840624 410 136.840624 307.72547 136.840624 195.456869 136.840624 111.344795 0 111.291407 0 0"></polygon>
                    <rect id="grey-strip" fill-opacity="0.478147645" fill="#55ACAA" fill-rule="nonzero" x="127" y="201" width="158" height="112"></rect>
                </g>
            </g>
        </g>
    </g>
</svg>
```

Reply:
i think it should because `style` is just an element here with rules as innerHTML. i need to check it though to say for sure

Reply:
I am more concerned with `/* <![CDATA[ */`; other than that I also think that it will work.